x/swaprouter/types/msg_create_pool.go

```diff
@@ -13,5 +19,13 @@ type CreatePoolMsg interface {
 	// Initial Liquidity for the pool that the sender is required to send to the pool account
 	InitialLiquidity() sdk.Coins
 	// CreatePool creates a pool implementing PoolI, using data from the message.
-	CreatePool(ctx sdk.Context, poolID uint64) (TraditionalAmmInterface, error)
+	CreatePool(ctx sdk.Context, poolID uint64) (gammtypes.TraditionalAmmInterface, error)
```

Comment:
Would it be possible to have TraditionalAMMInterface live inside the swap router module?

Reply:
This is the goal of a future refactor